plugins/modules/monitoring/uptimerobot.py

```diff
@@ -8,138 +8,178 @@
 
 
 DOCUMENTATION = '''
-
 module: uptimerobot
-short_description: Pause and start Uptime Robot monitoring
+short_description: Create, remove, pause and start monitors in Uptimerobot
 description:
-    - This module will let you start and pause Uptime Robot Monitoring
-author: "Nate Kingsley (@nate-kingsley)"
+    - This module lets you create, remove, pause and start monitors in Uptimerobot
+author:
+- "Nate Kingsley (@nate-kingsley)"
+- "Ninjiner (@Ninjiner)"
+version_added: "1.9"
```

Comment:
```suggestion
```

Reply:
Changed in: https://github.com/ansible-collections/community.general/pull/961/commits/1b23056c9a8000f9bf8dc3063ca137bd9d179ccf